src/js_native_api_v8.cc

```diff
@@ -2612,8 +2617,16 @@ napi_status napi_create_external_arraybuffer(napi_env env,
   CHECK_ARG(env, result);
 
   v8::Isolate* isolate = env->isolate;
+  std::unique_ptr<v8::BackingStore> backing =
+      v8::ArrayBuffer::NewBackingStore(external_data,
+                                       byte_length,
+                                       [](void*, size_t, void*){},
+                                       nullptr);
```

Comment:
Same suggestion as in `node_buffer.cc`, maybe add a comment that the `ArrayBufferReference` below takes care of the cleanup?

Reply:
fixed